README.md

```diff
@@ -188,31 +188,31 @@ to get started with Vagrant, please see [CONTRIBUTING.md](CONTRIBUTING.md).*
 #### Publishing metrics to a file
 To start collecting Mesos metrics and publish them to a file, you'll need to perform the following steps.
 
-
-Start the Snap daemon in the background:
+Set up the [Snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started),
+in one terminal window, run `snapd` (in this case with logging set to 1, trust disabled and global configuration saved in snap-config-example.json):
 
 ```
 $ snapd --plugin-trust 0 --log-level 1 --config examples/configs/snap-config-example.json \
     > /tmp/snap.log 2>&1 &
 ```
 
-Assuming you're in the working directory for this plugin, load the Mesos collector plugin:
-
+In another terminal window, download and load [snap-plugin-publisher-file](https://github.com/intelsdi-x/snap-plugin-publisher-file):
 ```
-$ snapctl plugin load build/rootfs/snap-plugin-collector-mesos
+$ wget http://snap.ci.snap-telemetry.io/plugins/snap-plugin-publisher-file/latest/linux/x86_64/snap-plugin-publisher-file
```

Comment:
I'd prefer that this used releases that are published to GitHub and not the latest build from CI.